web/src/lib/components/shared-components/side-bar/side-bar-button.svelte

```diff
@@ -22,7 +22,8 @@
     {isSelected
 		? 'bg-immich-primary/10 dark:bg-immich-dark-primary/10 text-immich-primary dark:text-[#adcbfa] hover:bg-immich-primary/25'
 		: ''}
-		{isCollapsed ? 'pl-5' : 'px-5'}
+		max-xs:pl-5
+		{isCollapsed ? 'pl-5' : 'xs:px-5'}
   "
```

Comment:
The JS based breakpoint `isCollapsed` and the CSS `xs` breakpoint are starting to get mixed here. I think it would be better to stick with either one. Using multiple breakpoint systems can lead to confusion and make debugging errors harder.

Reply:
Changed in 607c55e4c6d6ea63a4e118f76f64b1d8f1231bad